### ISSUE TITLE
Unable to reach Plex

### ISSUE THREAD

**Issue by ASchluterman** [TechPerplexed/Gooby#180]
Hello there! 

First I want to say thank you for running and maintaining Gooby. I have been using Gooby to run my VPS Plex server for over three years now. In that time I have rebuilt the servers a couple times, but recently am stumbling with getting it off the ground. 

A little background: I am changing VPS providers to Hetzner, with domain registration through Namecheap and DNS running through Cloudfare. Gooby is packaged well enough for me to implement and understand (well done!), however, my experience with Debian and resolving issues is nonexistent. I am unsure if my current issue is with my server or domain host/Cloudfare.

Following the walkthough on TechPerplexed, everything works as normal up until the Plex installation. The installation runs, grab my claim key, and then attempt to head over to plex.mydomain.site:8443. It is here where it is unable to communicate with the server and times out. Initially I have been receiving Error Code 522, which I believe is a Cloudfare error. After doing some research and changing settings, I am now getting a 400 Bad Request error.

I am a bit stumped and am outside of my depths. Cloudfare support is telling me to talk with my domain provider, and my domain provider is telling me to talk to my DNS provider.

I greatly appreciate your time and thank you for any advice you might be able to provide. Any suggestions?

**Comment by TechPerplexed** [comment]
Have you tried just to visit plex.yourdomain? Does plex.tv see your new server?

**Comment by ASchluterman** [comment]
Yep! I have tried to visit both versions of my domain with just the A Record and then with the 8443 port. Neither turn over.

I tried two full rebuilds over the weekend without success, all ending at the same spot - the Plex install. 

Even going to Plex.tv, nothing is recognized.

**Comment by TechPerplexed** [comment]
Hmm, I can't think of anything right off the bat... what happens if you try it with the CloudFlare proxy turned off (at least until it works)?

**Comment by ASchluterman** [comment]
I've tried that and another DNS service as well without success.

**Comment by TechPerplexed** [comment]
Can you post the exact steps you're taking to get Plex going? I will try to reproduce the issue.

**Comment by ASchluterman** [comment]
Sure.

Request rebuild of server (Debian 10.10 Minimal)
![Screen Shot 2021-10-02 at 10 29 56 AM](https://user-images.githubusercontent.com/51297624/135726084-8555e045-4fc2-42f1-b4d4-ff96e7a3277c.png)

Then run through the following string:
apt update
apt upgrade
apt install sudo
sudo wget http://bit.ly/GetGooby -O /tmp/install.sh && sudo bash /tmp/install.sh

Then I follow the tutorial on Techperplexed
Gooby
Create new user
Enter my domain
Enter my email
Install of Rclone (works fine)
Install of Plex (where it goes off the rails)
Am able to grab the claim key and install says it completes. When I try to claim my server, nothing responds nor is a new server recognized to be claimed.

I'll see if I can record the process if that would be helpful?

**Comment by TechPerplexed** [comment]
Right, well, just mention the steps exactly "where it goes off the rails" - I trust you with everything before that 😄
